src/Database/TypeFactory.php

```diff
@@ -53,6 +53,7 @@ class TypeFactory
         'timestampfractional' => Type\DateTimeFractionalType::class,
         'timestamptimezone' => Type\DateTimeTimezoneType::class,
         'uuid' => Type\UuidType::class,
+        'enum' => Type\EnumType::class,
```

Comment:
Wasn't sure if this is actually needed anymore but left it here.

Reply:
This isn't necessary as the enum type can't function without a enum class being set for it.